packages/ember-data/tests/unit/record-arrays/filtered-record-array-test.js

```diff
@@ -0,0 +1,13 @@
+var filteredArray;
+
+module("unit/record-arrays/filtered-record-array - DS.FilteredRecordArray", {
+  setup: function() {
+    filteredArray = DS.FilteredRecordArray.create({type: 'recordType'})
```

Comment:
linter is angry here, `DS.FilteredRecordArray.create({ type: 'recordType' })` (missing spaces)